### DISCUSSION TITLE
wallpaper centered

### DISCUSSION THREAD

**Discussion by senhor-R** [linuxdeepin/developer-center#2964] [Q&A | 问答板块]
help me, how to change wallpaper size like centered.
what config file modify

sorry my bad english.

**Comment by BLumia** [comment]
Sorry for the super late response since we did not look into issues that were created outside the developer-center repo.

I'm not sure if I understand it correctly, but the wallpaper is centered by default. You can set wallpaper in the file manager app's context menu, choose `Set as wallpaper` :)
